### ISSUE TITLE
Unable to remove user from team: 404 & 500 Error

### ISSUE THREAD

**Issue by davideperozzi** [umami-software/umami#2036]
### Describe the Bug

I'm currently checking out the self-hosted version of umami and already love it! 
One thing I've noticed is that I can't remove a user after I've created him/her.
Here's what I did:

- Created a team
- Created a user
- Attached the user to the team 
- Tried to delete the user again -> 500 Error (I've attached the server logs)
   - In the frontend it didn't really say anything despite "500 Error"
- After that I've tried to remove him from the team. Now I get a 404 Error in the frontend (no server error)

I'm using the latest umami docker image

### Database

PostgreSQL

### Relevant log output

```shell
Invalid `prisma.user.delete()` invocation:
The change you are trying to make would violate the required relation 'TeamUserToUser' between the `TeamUser` and `User` models.
    at pn.handleRequestError (/app/node_modules/@prisma/client/runtime/library.js:176:6477)
    at pn.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/library.js:176:5907)
    at pn.request (/app/node_modules/@prisma/client/runtime/library.js:176:5786)
    at async t._request (/app/node_modules/@prisma/client/runtime/library.js:179:10484) {
  code: 'P2014',
  clientVersion: '4.13.0',
  meta: {
    relation_name: 'TeamUserToUser',
    model_a_name: 'TeamUser',
    model_b_name: 'User'
  }
}
```

### Which browser are you using? (if relevant)

Chrome

### How are you deploying your application? (if relevant)

Docker (Swarm)

**Comment by franciscao633** [comment]
I was able to reproduce this. Labelling as bug. Thanks for reporting!

**Comment by franciscao633** [comment]
Fixed in v2.3.0
